app/presenters/v3/space_usage_summary_presenter.rb

```diff
@@ -0,0 +1,42 @@
+require 'presenters/v3/base_presenter'
+
+module VCAP::CloudController::Presenters::V3
+  class SpaceUsageSummaryPresenter < BasePresenter
+    def to_hash
+      {
+        usage_summary: {
+          started_instances: started_instances,
+          memory_in_mb: space.memory_used,
+          routes: space.routes_dataset.count,
+          service_instances: space.number_service_instances,
```

Comment:
Maybe the method `number_service_instances` is not necessary - could simply be `space.service_instances_dataset.count`?